.eslintrc

```diff
@@ -16,7 +16,7 @@
 
   "rules": {
     "react/prop-types": 0,
-    "@typescript-eslint/no-unused-vars": ["error", {"varsIgnorePattern": "_"}],
```

Comment:
We should always provide a reason when changing the rules: This is very useful when you want to remove properties from an object by destructuring. I.e.

```typescript
const {name, ...dataWithOutName} = contact;
```
You don't need the name here, but need to state it explicitly